components/org.wso2.carbon.identity.scim2.provider/src/main/java/org/wso2/carbon/identity/scim2/provider/resources/GroupResource.java

```diff
@@ -394,6 +395,9 @@ public Response patchGroup(@PathParam(SCIMConstants.CommonSchemaConstants.ID) St
         requestAttributes.put(SCIMProviderConstants.ATTRIBUTES, attribute);
         requestAttributes.put(SCIMProviderConstants.EXCLUDE_ATTRIBUTES, excludedAttributes);
         requestAttributes.put(SCIMProviderConstants.SEARCH, "0");
+
+        SupportUtils.updateIdentityContextFlow(Flow.Name.USER_GROUP_UPDATE);
```

Comment:
Should this be user group update or just group update ?
user group sorts of give a feeling it the group of the user

Reply:
Fixed deea09634a5600968d9a6650c8f6ff853cf60586